VagrantBuild.sh

```diff
@@ -38,7 +38,15 @@ fi
 
 echo "Building Hoot... "
 echo "Will take several extra minutes to build the training data the initial time Hootenanny is installed only."
-make -s clean && make -sj$(nproc)
```

Comment:
Talking with the team, we wonder if perhaps we should leave the original make invocation knowing that it's currently failing for CentOS 7x.  Then after we get this change set merged in, we can work on figuring out why the CentOS 7 make is failing on the first run.

Reply:
SGTM, I'll revert this tomorrow.